### PR TITLE
Dev- About Us Page

### PR DESCRIPTION
**Pull request by eringurley**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="src/style.css">
+    <title>Document</title>
```
Comment (tmzaragoza) [inline index.html]: Add a title to your document

File: index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    <h1>About RealCoolKids</h1>
+    <p>These are the RealCoolKids, and this is a list of their favorite things.</p>
+    <ul>
+        <li><a href="#Jack">Jack</a></li>
+        <li><a href="#Denise">Denise</a></li>
+        <li><a href="#Erin">Erin</a></li>
+        <li><a href="#Leigh-Ann">Leigh-Ann</a></li>
+    </ul>
+    <hr>
+    <div id="Jack">
```
Comment (Katedam) [inline index.html]: Use a semantic section or article tag here instead of a div

File: index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        <h2>Jack</h2> 
+        <img src="assets/skiing.jpg" alt="waterskiing">
```
Comment (Katedam) [inline index.html]: consider making your alt more descriptive - a screen reader will use this to describe the image to visually impaired individuals.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        <img src="assets/neon.jpeg" alt="neon">
+        <p>I studied glass bending in college.</p>
+        <h3>These are my favorite things:</h3>
+        <ol>
+            <li>eurovision</li>
+            <li>puppies</li>
+            <li>podcasts</li>
+        </ol>
+    </div>
+
```
Comment (Katedam) [inline index.html:68]: A footer tag is always a good idea! <footer>By Leigh-Ann, Erin, Denise, & Jack</footer> <- include links to your GitHub?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="src/style.css">
+    <title>About RealCoolKids</title>
+</head>
+<body>
+    <h1>About RealCoolKids</h1>
```
Comment (Katedam) [inline index.html]: This is a simple page so not entirely necessary but it's a good idea to get into the habit of putting your header text/images/sometimes nav? in a <header></header> tag for semantics.

**Pull request merged**